client/main.lua

```diff
@@ -259,15 +259,15 @@ end)
 
 RegisterNUICallback('close', function(_, cb)
     setCityhallPageState(false, false)
-    if not Config.UseTarget and inRangeCityhall then exports['qb-core']:DrawText('[E] Open Cityhall') end -- Reopen interaction when you're still inside the zone
+    if not Config.UseTarget and inRangeCityhall then exports['qb-core']:DrawText(Lang:t('info.city_services_menu')) end -- Reopen interaction when you're still inside the zone
     cb('ok')
 end)
 
 RegisterNUICallback('requestId', function(id, cb)
     local license = Config.Cityhalls[closestCityhall].licenses[id.type]
     if inRangeCityhall and license and id.cost == license.cost then
         TriggerServerEvent('qb-cityhall:server:requestId', id.type, closestCityhall)
-        QBCore.Functions.Notify(('You have received your %s for $%s'):format(license.label, id.cost), 'success', 3500)
+        QBCore.Functions.Notify((Lang:t('success.recived_license')):format(license.label, id.cost), 'success', 3500)
```

Comment:
recived is not correct. Should be 'received'. This is wrong throughout the PR. 

And I also agree that this change does not really fit with the format we currently have. Why does this need to be implemented?